libsolidity/codegen/MultiUseYulFunctionCollector.h

```diff
@@ -41,6 +41,11 @@ class MultiUseYulFunctionCollector
 	/// cases.
 	std::string createFunction(std::string const& _name, std::function<std::string()> const& _creator);
```

Comment:
At some point we can delete this one. 

Then we can have `m_requestedFunctions` to actually map from name to a struct that contains code, number of input and output params, as both of these would be available in new function. Than we will have that information over here, and we would not need to pass number of params with `callYulFunction`